code/datums/supplypacks/contraband_vr.dm

```diff
@@ -4,4 +4,16 @@
 	cost = 150
 	containertype = /obj/structure/closet/crate
 	containername = "Stolen crate"
-	contraband = 1
\ No newline at end of file
+	contraband = 1
+
+/datum/supply_packs/supply/awoo
+	name = "Wolfgirl Crate"
+	cost = 200 //I mean, it's a whole wolfgirl
+	containertype = /obj/structure/largecrate/animal/awoo
+	containername = "Wolfgirl crate"
+
+/datum/supply_packs/supply/mlem
+	name = "Catgirl Crate"
+	cost = 200 //I mean, it's a whole catgirl
+	containertype = /obj/structure/largecrate/animal/mlem
+	containername = "Catgirl crate"
```

Comment:
These should both be contraband.

Reply:
Do you mean Line 18, the Containertype line? `/obj/structure/largecrate/animal/mlem` is in that path because the held_type variable that determines that the crate holds a mob is defined in /obj/structure/largecrate/animal/. I can move it, but it'd have to be redefined in the new list.

Or did you mean the name variable?